components/brave_vpn/common/features.cc

```diff
@@ -28,7 +29,12 @@ BASE_FEATURE(kBraveVPNLinkSubscriptionAndroidUI,
 #if BUILDFLAG(IS_WIN)
 BASE_FEATURE(kBraveVPNDnsProtection,
              "BraveVPNDnsProtection",
-             base::FEATURE_ENABLED_BY_DEFAULT);
+#if BUILDFLAG(ENABLE_BRAVE_VPN_SERVICE)
```

Comment:
AFAIK, you want to use this buildflag for dns filter service.
But, This feature `BraveVPNDnsProtection` also affects `Enable DoH for Brave VPN` behavior.
Do you want to disable it together by using this buildflag?

Reply:
Both the service and the feature do the same job of protecting users from DNS leaks, we don't need to enable them both at the same time. If we have a service need to disable the feature

Reply:
Oops, I thought this feature is also disabled when buildflag is disabled.
Make sense!